c++/src/Common.cc

```diff
@@ -131,8 +131,11 @@ namespace orc {
   }
 
   std::string FileVersion::toString() const {
+    if (majorVersion == 1 && minorVersion == 9999) {
+      return "UNSTABLE-PRE-2.0";
```

Comment:
Can we also skip writing & reading scale stream for Decimal128 in UNSTABLE-PRE-2.0?

Reply:
I think yes, but it'd be better to use RLEv3 since the values might have more tailing zeros than what we observed in https://github.com/apache/orc/pull/257#issuecomment-385100087